components/main.css

```diff
@@ -0,0 +1,3 @@
+.container {
+  width: 600px;
```

Comment:
let us just do css inside js file just like the one in main.js line 7.

Reply:
Ok I forgot to delete that CSS file. I don't believe I referenced it anywhere so that should be fine